packages/stepper/src/index.ts

```diff
@@ -0,0 +1 @@
+export { default, Step } from './Stepper';
```

Comment:
My stance on compound components is that they should all be named exports.

My opinion (so take with a grain of salt) is that a default export should be expected to work by itself. 

That's not the case here, these components need to work together, hence they should all be named exports.

Happy to discuss more!